intelmq/bots/parsers/dataplane/parser.py

```diff
@@ -50,23 +50,29 @@ class DataplaneParserBot(ParserBot):
     }
 
     def parse_line(self, line, report):
+        file_format = [
```

Comment:
It occurs to me now that this variable could be extracted to `__init__`  in order to save the variable allocations.